docs/Submodules.md

```diff
@@ -101,3 +101,9 @@ $GitPromptSettings.EnableFileStatus = $false
 ```
 
 You can disable this permanently by adding to your `$PROFILE` file. (`notepad $PROFILE`)
+
+## Error messages
```

Comment:
We can include a new section in https://github.com/dotnet/aspnetcore/blob/main/docs/BuildErrors.md#build-errors and clarify how to resolve it by running `git submodule update --init --recursive`